3rd-party/treematch/tm_topology.c

```diff
@@ -8,26 +12,27 @@
 
 
 tm_topology_t* tm_get_local_topo_with_hwloc(void);
-tm_topology_t* hwloc_to_tm(char *filename);
-int int_cmp_inc(const void* x1,const void* x2);
-void optimize_arity(int **arity, double **cost, int *nb_levels,int n);
-int symetric(hwloc_topology_t topology);
-tm_topology_t * tgt_to_tm(char *filename);
+static tm_topology_t* hwloc_to_tm(char *filename);
+int treematch_int_cmp_inc(const void* x1,const void* x2);
+static void optimize_arity(int **arity, double **cost, int *nb_levels,int n);
+static int symetric(hwloc_topology_t topology);
+static tm_topology_t * tgt_to_tm(char *filename);
 void tm_display_arity(tm_topology_t *topology);
```

Comment:
There seems to have been the namespace the namespace `tm_*` already but all renaming here uses `treematch_*`. I don't care either way but maybe it should be consistent?

Reply:
Renamed all treematch_ symbols to tm_